docker/scripts/bump_sdk_v0_version.sh

```
@@ -0,0 +1,528 @@
+    # 3. Update the unstract-sdk version in the tool's requirements.txt
+    if [[ -f "$requirements_file" ]]; then
+        echo "Updating SDK version in $tool_name requirements.txt"
+        update_sdk_version "$requirements_file" "$sdk_version_arg"
+    fi
+}
+
+# Update SDK version in pyproject.toml or requirements.txt
+update_sdk_version() {
+    local file="$1"
+    local version_arg="$2"
+    local dir_name=$(dirname "$dir")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)